adds config variables

angusmcgee opened the pull request:
adding picklist fields for address MAL, id tag for apex:form, MAL 9, 10
and 11.

Commit:
adds config variables

adding picklist fields for address MAL, id tag for apex:form, MAL 9, 10
and 11.

Commit:
Merge branch 'master' into Adds-Config-Variables

noncorpus commented on src/pages/LookupConfig.page:
```diff
@@ -1,3 +1,44 @@
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "County Field" for = "county_field"/>
+					<apex:selectList value = "{!countyField}" multiselect = "false" size = "1" id = "county_field">
+					</apex:selectList>
```
We'll need to add a `<apex:selectOptions>` tag here (similar to line 13, but using the `fieldList` variable instead of `objectList`.  Additionally, since we don't have those values before an object is chosen, we'll need to set the initial value of the `fieldList` variable to equal to a `new List<SelectOption>()` in the constructor of the controller.  You could easily put it right before you call `populateObjectList()` on line 14 of the controller.

noncorpus commented on src/pages/LookupConfig.page:
```diff
@@ -1,3 +1,44 @@
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "State Field" for = "state_field"/>
+					<apex:selectList value = "{!stateField}" multiselect = "false" size = "1" id = "state_field">
+					</apex:selectList>
```
Same as above.

noncorpus commented on src/pages/LookupConfig.page:
```diff
@@ -1,3 +1,44 @@
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "Street Address Field" for = "street_address_field"/>
+					<apex:selectList value = "{!streetField}" multiselect = "false" size = "1" id = "street_address_field">
+					</apex:selectList>
```
And again...

noncorpus commented on src/pages/LookupConfig.page:
```diff
@@ -1,3 +1,44 @@
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "City Field" for = "city_field"/>
+					<apex:selectList value = "{!cityField}" multiselect = "false" size = "1" id = "city_field">
+					</apex:selectList>
```
Another one (at least you'll be able to copy/paste the line of code each time 😄 )

noncorpus commented on src/pages/LookupConfig.page:
```diff
@@ -1,3 +1,44 @@
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "Zip Code Field" for = "zip_code_field"/>
+					<apex:selectList value = "{!zipField}" multiselect = "false" size = "1" id = "zip_code_field">
+					</apex:selectList>
```
Last one...

noncorpus reviewed (changes requested):
This is great work!  Other than my one comment (that applies 5 times), this can be merged with Master.  Feel free to tackle the changes to address the comment(s), and we can sync up tomorrow on adding those additional changes to the branch and this PR.

runicfox committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Adds-Config-Variables

runicfox committed:
Adds Config Variables

Need to commit changes to LookupConfigController class as well as
LookupConfig page.
